WordPress/src/main/java/org/wordpress/android/util/FormattableContentUtils.kt

```diff
@@ -0,0 +1,70 @@
+package org.wordpress.android.util
+
+import org.wordpress.android.fluxc.tools.FormattableContent
+import org.wordpress.android.fluxc.tools.FormattableMedia
+import org.wordpress.android.fluxc.tools.FormattableRange
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class FormattableContentUtils @Inject constructor() {
```

Comment:
I think this would be a great place for extension functions in a file instead of injected class. You can still write a `FormattableContentTest` to test them and you probably want the actual implementation in tests instead of mocking it all.

Reply:
Good idea! ;)